requirements.txt

```diff
@@ -0,0 +1,6 @@
+django-nose==1.4.7
```

Comment:
Where is this used? I noticed you added the file, but not the install command?

I also noticed you install flake manually?